ps/jenkins/package-testing-ps-8.0.yml

```diff
@@ -0,0 +1,20 @@
+- job:
+    name: package-testing-ps80
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    properties:
+        - build-discarder:
+            artifact-days-to-keep: -1
+            artifact-num-to-keep: 10
+            days-to-keep: -1
+            num-to-keep: 10
+    pipeline-scm:
```

Comment:
Parameters defined in .groovy file should be defined here as well https://github.com/Percona-Lab/ps-build/blob/5.7/jenkins/pipeline.yml#L16

This should be implemented across all pushed .yml files

Reply:
Done, thank you 👍